Symfony/CS/Tests/Fixer/Symfony/MultilineArrayTrailingCommaFixerTest.php

```diff
@@ -298,7 +260,78 @@ function foo(array $a)
         "string"
         /* foo */);',
             ),
-            array(
+        );
+    }
+
+    /**
+     * @dataProvider provideExamples54
+     * @requires PHP 5.4
+     */
+    public function testFix54($expected, $input = null)
+    {
+        $this->makeTest($expected, $input);
+    }
+
+    public function provideExamples54()
+    {
+        return array(
+                // short syntax tests
+                array('<?php $x = array([]);'),
+                array('<?php $x = [[]];'),
```

Comment:
yep, it would be changes it.
most of changes, as in your prev PR, are good. but the short_open_tag fixer is great example to not change everything blindly.